contrib/clojure-package/src/dev/generator.clj

```diff
@@ -470,6 +476,53 @@
   (println "Generating symbol-api file")
   (write-to-file all-symbol-api-functions symbol-api-gen-ns "src/org/apache/clojure_mxnet/gen/symbol_api.clj"))
 
+;;;;;;; SymbolRandomAPI
+
+(defn gen-symbol-random-api-function [op-name]
+  (let [{:keys [fn-name fn-description args]} (gen-op-info op-name)
+        fn-name (remove-prefix "-random-" fn-name)
```

Comment:
the scala package does it for both `random_` and `sample_` (see: https://github.com/apache/incubator-mxnet/blob/master/scala-package/macros/src/main/scala/org/apache/mxnet/GeneratorBase.scala#L180).

Reply:
Do we also want a `ndarray-sample` namespace or should we use a prefix like `sample-op-name` in the `ndarray-random` namespace?